src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDepositSpec.scala

```diff
@@ -578,30 +578,31 @@ class AddDatasetMetadataToDepositSpec extends UnitSpec with BeforeAndAfterAll {
       .map(dss => dss.map { case (_, ds) => AddDatasetMetadataToDeposit.datasetToXml(ds) })
   }
 
-  "createDcmiMetadata" should "return the expected spatial elements" in {
+  "createDcmiMetadata" should "return the expected spatial Box elements" in {
     val dataset = new Dataset() +=
       "DCT_SPATIAL" -> List("here", "there", "", "") +=
       "DCX_SPATIAL_SCHEME" -> List("degrees", "RD", "", "") +=
-      "DCX_SPATIAL_X" -> List("83575.4", "", "", "") +=
-      "DCX_SPATIAL_Y" -> List("455271.2", "", "", "") +=
-      "DCX_SPATIAL_NORTH" -> List("", "1", "", "") +=
-      "DCX_SPATIAL_SOUTH" -> List("", "2", "", "") +=
-      "DCX_SPATIAL_EAST" -> List("", "3", "", "") +=
-      "DCX_SPATIAL_WEST" -> List("", "4", "", "")
+      "DCX_SPATIAL_NORTH" -> List("4", "40", "", "") +=
+      "DCX_SPATIAL_SOUTH" -> List("3", "30", "", "") +=
+      "DCX_SPATIAL_EAST" -> List("2", "20", "", "") +=
+      "DCX_SPATIAL_WEST" -> List("1", "10", "", "")
     val expectedXml = <ddm>
       <ddm:dcmiMetadata>
         <dcterms:spatial>here</dcterms:spatial>
         <dcterms:spatial>there</dcterms:spatial>
-        <dcx-gml:spatial srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
-          <Point xmlns="http://www.opengis.net/gml">
-            <pos>455271.2 83575.4</pos>
-          </Point>
+        <dcx-gml:spatial>
+          <boundedBy xmlns="http://www.opengis.net/gml">
+            <Envelope srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
+              <lowerCorner>3 1</lowerCorner>
+              <upperCorner>4 2</upperCorner>
+            </Envelope>
+          </boundedBy>
         </dcx-gml:spatial>
         <dcx-gml:spatial>
           <boundedBy xmlns="http://www.opengis.net/gml">
             <Envelope srsName="http://www.opengis.net/def/crs/EPSG/0/28992">
-              <lowerCorner>1 3</lowerCorner>
-              <upperCorner>2 4</upperCorner>
+              <lowerCorner>10 30</lowerCorner>
+              <upperCorner>20 40</upperCorner>
```

Comment:
mmm, is it right that the values for `lowerCorner` and `upperCorner` are swapped in comparison with the one above and the input?